code/extensions/che-remote/package.nls.json

```diff
@@ -5,6 +5,7 @@
 	"openDashboardCommand": "Eclipse Che: Open Dashboard",
 	"openOpenShiftConsoleCommand": "Eclipse Che: Open OpenShift Console",
 	"stopWorkspaceCommand": "Eclipse Che: Stop Workspace",
+	"stopWorkspaceAndRedirectToDashboard": "Eclipse Che: Stop Workspace and Redirect to Dashboard",
```

Comment:
please provide the corresponding changes for the devspaces-code assembly, something like: https://github.com/redhat-developer/devspaces-images/pull/409/files

Reply:
also, it would be nice to update `Branding` section, something similar to https://github.com/che-incubator/che-code/pull/190/files

Reply:
I will merge it after this PR - https://github.com/redhat-developer/devspaces-images/pull/507